src/DSE.Open.Observations/BinarySpeechSoundObservation.cs

```diff
@@ -40,7 +40,7 @@ public static BinarySpeechSoundObservation Create(
     [SkipLocalsInit]
     protected override ulong GetDiscriminatorId()
     {
-        var chars = SpeechSound.ToCharSequence().AsSpan();
+        var chars = SpeechSound.ToString();
```

Comment:
`SpeechSymbol` implements `IUtf8SpanSerializable` - we could carry on up and have `SpeechSound` also implement it - then this could use a stack allocated buffer.